week1/HomeWork/index.js

```diff
@@ -0,0 +1,32 @@
+let http = require('http');
+const port = 3000;
+var state = 10; 
+
+let server = http.createServer( (request, response) => {
+
+   if(request.url === "/state") {
+	   response.write(String(state));
+   }
+   else if(request.url === "/add") {
+	   state++;
+	   response.write(String(state));
+   }
+   else if(request.url === "/remove") {
+	   state--;
+	   response.write(String(state));
+   }
+   else if(request.url === "/reset") {
+	   state = 10;
+	   response.write(String(state));
+   }
+   else{
+	   response.write("Not found! , Please check the url path");
```

Comment:
Nice job here - for a 'page not found', we probably want to return with an HTTP status code of 404. Can you adjust it so that it returns with a status code of 200 for successful requests, and a 404 for the 'not found' page?